ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -247,97 +248,101 @@ static GHashTable *tizensensors = NULL;
-    { .type=SENSOR_PROXIMITY, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_PRESSURE, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_ULTRAVIOLET, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_TEMPERATURE, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HUMIDITY, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HRM, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_INT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HRM_LED_GREEN, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_INT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HRM_LED_IR, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_INT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HRM_LED_RED, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_INT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_GYROSCOPE_UNCALIBRATED, .value_count=6,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={6, 1, 1, 1} }},
-    { .type=SENSOR_GEOMAGNETIC_UNCALIBRATED, .value_count=6,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={6, 1, 1, 1} }},
-    { .type=SENSOR_GYROSCOPE_ROTATION_VECTOR, .value_count=4,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={4, 1, 1, 1} }},
-    { .type=SENSOR_GEOMAGNETIC_ROTATION_VECTOR, .value_count=4,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={4, 1, 1, 1} }},
-    { .type=SENSOR_SIGNIFICANT_MOTION, .value_count=1,
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HUMAN_PEDOMETER, .value_count=8, /* Last 5 values might be flost32..? */
-      .tinfo = { .name="values", type=_NNS_INT32,
-                 .dimension={8, 1, 1, 1} }},
-    { .type=SENSOR_HUMAN_SLEEP_MONITOR, .value_count=1, /* STATE */
-      .tinfo = { .name="values", type=_NNS_INT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HUMAN_SLEEP_DETECTOR, .value_count=1, /** @todo check! */
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_HUMAN_STRESS_MONITOR, .value_count=1, /** @todo check! */
-      .tinfo = { .name="values", type=_NNS_FLOAT32,
-                 .dimension={1, 1, 1, 1} }},
-    { .type=SENSOR_LAST, .value_count=0, .tinfo = {0, }},
+  {.type = SENSOR_ACCELEROMETER,.value_count = 3,
+        .tinfo = {.name = valstr,.type = _NNS_FLOAT32,
```

Comment:
Question: is it necessary to set name 'values' in GstTensorInfo? (NULL if possible)

Reply:
No. it's not. So removed :)